app/globals.py

```diff
@@ -83,6 +83,11 @@ def get_answer_store(user):
     return questionnaire_store.answer_store
 
 
+def get_list_store(user):
```

Comment:
not used

Reply:
Still used by _get_context()

Reply:
But that can be switched to the questionnaire_store